docs/user-guide/cli-using-usingcli.md

```diff
@@ -1,7 +1,35 @@
 # Using Zowe CLI
 
-Learn about how to use Zowe CLI, including connecting to the mainframe, managing profiles, integrating with API Mediation Layer, and more.
+In this section, learn about how to use Zowe CLI, including connecting to the mainframe, managing profiles, integrating with API Mediation Layer, and more.
 
 You can use the CLI interactively from a command window on any computer on which it is installed, or run it in a container or automation environment.
 
-**Tip:** If you want to use the CLI together with a screen reader to provide accessibility, we recommend using the Mac™ Terminal application enabled for Accessibility through [System Preferences > Accessibility](https://support.apple.com/zh-sg/guide/terminal/trml1020/mac). On Windows™, adjust the Properties settings in Command Prompt. For other operating systems, or for alternative terminals, check the specification for the terminal to ensure that it meets accessibility requirements.
\ No newline at end of file
+**Tip:** If you want to use the CLI together with a screen reader to provide accessibility, we recommend using the Mac™ Terminal application enabled for Accessibility through [System Preferences > Accessibility](https://support.apple.com/zh-sg/guide/terminal/trml1020/mac). On Windows™, adjust the Properties settings in Command Prompt. For other operating systems, or for alternative terminals, check the specification for the terminal to ensure that it meets accessibility requirements.
+
+## Supported operating systems and package/resource managers: 
+
+### Operating systems
+
+- MacOS 11 (Big Sur), 12 (Monterey)
+- Unix-like:
+   - [CentOS](https://www.centos.org/) 8+
+
+   - [RHEL](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) 8+
+   - [Ubuntu](https://ubuntu.com/) 20.04+
+
+- Windows 10+
+- [z/OS Unix Systems Services](https://www.ibm.com/docs/en/zos/2.4.0?topic=descriptions-zos-unix-system-services) <sup>*</sup>
+
+    <b>*</b> Zowe CLI does not support secure credential storage in this environment. 
```

Comment:
Question for the team: If a customer is ok with plain text creds, are we currently in a position to officially support everything else under USS? I did not think that we had reached that point.

Reply:
As far as I am aware, DB2 support should exist if the instructions in ibm_db are followed, which as far as I am aware, is not documented anywhere in the Zowe space. The daemon is also not going to work, and I do not think there is any way that we will be able to get that to work. Aside from those, the node CLI should work without any issues.

Reply:
My reason for this question is that there is a difference between "most stuff works" and "we support this environment". For example, are we prepared to accept and fix an issue reported on USS (even in the areas that we think should work) with an acceptable service level response time? Is USS part of our automated test suites? Are we confident that we are ready to support USS?

I added this question as a topic for the last Product Management meeting. However, the agenda was packed and we did not get to this question. The question remains on the agenda for the next meeting.

Reply:
Team consensus: Remove USS from list

To include USS in the future: Document install, complete list of items that don't work on USS (sec cred, daemon, etc.); internal testing functionality

Reply:
On this page (cli-using-usingcli.md), we could suggest that people who are interested in Zowe CLI on USS visit:  https://github.com/zowe/zowe-cli/issues/1680 and upvote (👍) to show interest.

Reply:
Feedback incorporated into the content. Thank you!